scraper.db.schema.sql

```diff
@@ -0,0 +1,55 @@
+
+CREATE TABLE layer_map (
```

Comment:
I wonder, how important is pageserver restarts for these stats? 
Would we want to add https://github.com/neondatabase/neon/pull/3527 to the dataset?

Reply:
That's what `launch_id` is. Will update the code to use the new name.